EnglishFilter/sections/general_extensions.txt

```diff
@@ -46,7 +46,7 @@ pornxday.com,putlockerslinks.com,unblockninja.blue,baixarhentai.net,freeservice.
 pornyhd.com,pornxday.com,putlockerslinks.com,unblockninja.blue,baixarhentai.net,freeservice.info,branitube.net,hindimean.com,sports-stream.net,btdb.nl,drobfilx.com,22pixx.xyz,nacastle.com,eurostreaming.pink,imgbaron.com,embed.media,feurl.com,zidiplay.com,animeshouse.biz,erai-ddl3.info,erai-ddl2.info,erai-ddl.info,hitbits.io,scnlog.me,plytv.me,premiersport.tv,mixdrop.to,streamcdn.to,300mbfilms.io,jkanime.net,lovetheboxx.be,uii.io,mixdrop.co,filma24.cc,anime-odcinki.pl,phim2online.com,upzone.cc,likn.xyz,t0r.xyz,sportstream.to,acefile.co,watchxxxfreeinhd.com,haloani.ru,premiumleecher.com,short.pe,fjav.net,kastream.biz,dropapk.to,aflamfree.top,khatrimazafull.to,mirrorace.com,kuntv.pw,egybest2.com,javplay.me,imgoutlet.pw,ytmp3.eu#%#//scriptlet("abort-on-property-read", "popjs")
 !
 ! PopAds-b
-javhiv.com,nyafilmer.com,gotgayporn.com,letwatch.to,hqq.tv,xonline.vip,pornhd.com,thevideo.me,sextfun.com,vidup.me,imgoutlet.com,imgrock.net#%#Object.defineProperty(window, 'popns', { get: function() { return {}; } });
+linkhub.icu,javhiv.com,nyafilmer.com,gotgayporn.com,letwatch.to,hqq.tv,xonline.vip,pornhd.com,thevideo.me,sextfun.com,vidup.me,imgoutlet.com,imgrock.net#%#Object.defineProperty(window, 'popns', { get: function() { return {}; } });
```

Comment:
I don't see `popns` even with US IP.

Reply:
Please use AdGuard for Linux, Mac or windows.

Reply:
I have [the same](https://github.com/AdguardTeam/AdguardFilters/pull/78261#discussion_r599583029). Why did you add the domain to this rule?
@AdamWr please take a look.

Reply:
Don't need him right now.

Reply:
@zloyden Ryan Brown added a rule which would fix the popup, so since it's integrated with Base Filter, it's fixed as for now.

Reply:
Ok.